### DISCUSSION TITLE
Welcome to pi-nexus-autonomous-banking-network Discussions!

### DISCUSSION THREAD

**Discussion by KOSASIH** [KOSASIH/pi-nexus-autonomous-banking-network#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TriCellsTechnologies** [comment]
Sir, is this really a decentralized application of Pi Network Official utility or it has nothing to do with Pi Network Official, if it is really an official function of Pi Network, then it is a very happy and big thing for Pi Network and we Pioneer.

**Comment by KOSASIH** [comment]
Yes,  i'm KOSASIH.  Thank you for your mail. May God blessing you. 🙏

KOSASIH

Pada tanggal Sab, 18 Mei 2024 09.19, Tri Cells Technologies <
***@***.***> menulis:

**Reply by tanxiaohai1** [reply]
Come on, I pay attention to you every day, and you have made a great contribution to the pie network

**Reply by qq3408** [reply]
牛逼克拉斯

**Comment by Hammadouch** [comment]
Thank you for the tremendous effort to make the Pay currency a success

**Comment by Clawue** [comment]
Selamat siang bapak...saya salah 1 penggemar bapak

**Comment by Hammadouch** [comment]
محفظتي مجمدة ..بدون ان اعمل اي شيء خارج القانون...

في الاثنين، 15 جويلية 2024, 6:36 ص Clawue ***@***.***> كتب:

**Comment by Hammadouch** [comment]
My wallet is frozen without me doing anything outside the law with it. I
will promise to return it.

في الاثنين، 15 جويلية 2024, 6:36 ص Clawue ***@***.***> كتب:

**Comment by TriThucViet2025** [comment]
> ## 👋 Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Hi Sir, have a nice day. I’m Khanh , I come from VietNam. I am very grateful you about contributions and efforts for development of Pi Network and for the project development “Pi Nexus autonomous banking network” . I pray for you and your family the best everything.
 About me, I’m trying to install “PiNexus banking “ to accelerate to Open Mainnet  and create a community that accepts pi in everyday life. I hope you and your team support for me to install that
 I tryed to do follow your guides and build Docker Image and run containers by use “Docker-compose.yml. But that’s error. 
 When “Run pip install -r equirements.txt “  in docker file to run that Its showed the error :
“ could don’t find the satisfactory file to run Web3.py==5.24.0 to run it “  and when pull equipments.txt so it error something. I don’t know how that are..
Sir can guide for me how to make truely . I hope that Sir will answer me soon .
Thank you very much.

**Reply by TriThucViet2025** [reply]
![Uploading image.jpg…]()

**Comment by Muiz1976** [comment]
🇮🇩 Bandung Jawa Barat satu propinsi.terima kasih banyak

**Comment by dai900611** [comment]
KOSASIH 先生 :非常感谢您对 Pi Network 开发和“Pi Nexus 自主银行网络”项目开发的贡献和努力。我为您和您的家人祈祷一切顺利~
